devicetypes/smartthings/aeon-multisensor.src/aeon-multisensor.groovy

```diff
@@ -85,12 +86,18 @@ metadata {
 		valueTile("battery", "device.battery", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
 			state "battery", label:'${currentValue}% battery', unit:""
 		}
-		standardTile("configure", "device.configure", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
-			state "configure", label:'', action:"configuration.configure", icon:"st.secondary.configure"
-		}
 
 		main(["motion", "temperature", "humidity", "illuminance"])
-		details(["motion", "temperature", "humidity", "illuminance", "battery", "configure"])
+		details(["motion", "temperature", "humidity", "illuminance", "battery"])
+	}
+
+	preferences {
```

Comment:
Because this will be visible to the user when they install it, I wonder if there's a better way to explain it to the user?  Thoughts @marstorp

Reply:
`		input description: "Please consult AEOTEC MULTISENSOR 6 operating manual for advanced setting options. You can skip this configuration to use default settings",
				title: "Advanced Configuration", displayDuringSetup: true, type: "paragraph", element: "paragraph"`

This explanation makes an appearance in the multisensor 6 DTH that I cribbed this from.

Reply:
Sounds like a reasonable description to me :)

Reply:
The code in configure where the variables are used checks to see if they exist first and uses defaults if not.

Reply:
yes, true but when you enter the preference page can you skip it or do you have to enter a value?

Reply:
Just checked on my site. It looks like the field is blank. If I want the defaults to be 20s and 5 minutes, should I use 

`defaultValue: "20 seconds"`

or 

`defaultValue: 1`

Reply:
From the [docs](http://docs.smartthings.com/en/latest/device-type-developers-guide/device-preferences.html?highlight=defaultValue):

> Setting a default value (defaultValue: "foobar") for an input may render that selection in the mobile app, but the user still needs to enter data in that field. It’s recommended to not use defaultValue to avoid confusion.

So the defaultValue field seems to be useless?

Reply:
Yeah... defaultValue is not really used properly (I need to see if there is a mobile ticket open for this). It really gets bad in Android if required:true is set. I think it might work in iOS (it did for me). Check out PROB-1673. If you look at the Aeon Siren code, it has a good use of enforcing defaultValue configuration and making sure that updated values are sent to the device.